assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -79,16 +97,31 @@ export default function SetupMainPAX( { finishSetup } ) {
 	const { setPaxConversionID, setExtCustomerID, submitChanges } =
 		useDispatch( MODULES_ADS );
 
-	const onCompleteSetup = useCallback( async () => {
-		if ( ! paxApp ) {
+	useMount( () => {
+		if ( PAX_SETUP_STEP.FINISHED === showPaxAppStep ) {
+			// If the PAX query param indicates the setup is finished on page load,
+			// set the step back to the PAX launch, as values are only temporarly
+			// saved in state after PAX campaign setup signal is received.
+			setShowPaxAppQueryParam( PAX_SETUP_STEP.LAUNCH );
+		}
+	} );
+
+	const onLaunch = ( app ) => {
+		if ( ! paxAppRef.current ) {
+			paxAppRef.current = app;
+		}
+	};
+
+	const onCampaignCreated = useCallbackOne( async () => {
```

Comment:
Why do we need `useCallbackOne` here instead of `useCallback`?

Reply:
Had a chat with Evan 2 days ago, he mentioned we might need ensure this callback does not cause re-render to the PAX app, so I used this approach to avoid instances where React might clear regular `useCallback` causing the re-render, as `useCallbackOne` will persist.

Reply:
Ah, yes, good point. 👍🏻